modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -32,46 +28,26 @@ nodes:
   - nodeCount: 2 <1>
     nodeSpec:
       resources: {}
-    roles:  <2>
-    - client
-    - data
-    - master
+    roles: 
+    - data <2>
 ----
 <1> Specify the number of nodes of this type.
-<2> Specify the type of node.
+<2> Specify the `data` role.
 +
-[NOTE]
-====
-You can create 1 or 3 master nodes.
-====
-+
-For example, to scale the data and master nodes:
+For example, to add an additional node:
```

Comment:
I think this is what we want to leave off and only encourage users on how to scale up their current deployment.
Thoughts @jcantrill ?